src/overlays/actors/ovl_Fishing/z_fishing.h

```diff
@@ -9,28 +9,26 @@ struct Fishing;
 typedef struct Fishing {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ char unk_14C[0x004];
-    /* 0x0150 */ u8 unk_150;
-    /* 0x0151 */ u8 unk_151;
+    /* 0x0150 */ u8 isLoach; //rename to fish type?
+    /* 0x0151 */ u8 unk_151; //lily byte
```

Comment:
`//comment` -> `// comment` in here

also what does "lily byte" mean? this seems fish-specific (and used for interacting with lilys somehow)